src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -76,26 +72,46 @@ public void run(RunConfiguration config) throws AppEngineException {
     }
 
     List<String> jvmArguments = new ArrayList<>();
-    if (appengineWeb.isJava8()) {
-      jvmArguments.add("-Duse_jetty9_runtime=true");
-      jvmArguments.add("-D--enable_all_permissions=true");
-    } else {
-      // Add in the appengine agent
-      String appengineAgentJar = new File(
-              sdk.getJavaAppEngineSdkPath().toFile(), "agent/appengine-agent.jar")
-              .getAbsolutePath();
-      jvmArguments.add("-javaagent:" + appengineAgentJar);
-    } 
     arguments.addAll(DevAppServerArgs.get("address", config.getHost()));
     arguments.addAll(DevAppServerArgs.get("port", config.getPort()));
     if (config.getJvmFlags() != null) {
       jvmArguments.addAll(config.getJvmFlags());
     }
+    arguments.addAll(DevAppServerArgs.get("default_gcs_bucket", config.getDefaultGcsBucketName()));
+    // TODO: It's not really clear what the behavior here is when this is specified
+    arguments.addAll(DevAppServerArgs.get("runtime", config.getRuntime()));
```

Comment:
It does some internal stuff in the devappserver. It seems "kickstart" code and devappserver code kind of overlap is some way that I don't understand. @ludoch ?